src/qt/rpcconsole.cpp

```diff
@@ -462,13 +462,19 @@ void RPCConsole::clear()
     }
 
     // Set default style sheet
+    QFontInfo fixedFontInfo(GUIUtil::fixedPitchFont());
+    // Try to make fixed font adequately large on different OS
+    QString ptSize = QString("%1pt").arg(QFontInfo(QFont()).pointSize() * 8.5 / 9);
```

Comment:
I also preferred to non-`* 8.5 / 9` solution. But i have ACKed this solution because it might slightly reduce the size of mostly-oversized monospace fonts. But a `*0.95` instead a * 8.5/9 would be cleaner, right.